### PR TITLE
Fixed #955: Init state and life circle for sign modal component

### LINKED ISSUE #955: fix src/shared/wallet/sign-and-send-envelop-modal.tsx
1. if user open the desktop wallet, then receive an envelop to sign, and then unlock the wallet. the page will be broken because you probably need `  public state: { envelop: { [key: string]: any } } = {envelop: {}};`

2. `shouldComponentUpdate` is always executing and then makes too many requests, and actually, it is always making requests. Please fix this.

### PR DESCRIPTION
**Pull request by nmhung1210**
**What type of PR is this?**
bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #955 

**Special notes for your reviewer**:
The `wss` seems not working even accepted the certs warning. This PR is tested with `ws` instead.

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by puncsky** [comment]
there are still problems with this workflow

1. open wallet

Screen Shot 2019-08-18 at 12 21 50 AM

2. call contract in https://member.iotex.io/tools/ERC-20

Screen Shot 2019-08-18 at 12 23 47 AM

3. continue to unlock the wallet. create a new wallet or import one.

Screen Shot 2019-08-18 at 12 21 34 AM

the values above are incorrect. they should be like

Screen Shot 2019-08-18 at 12 21 42 AM

**Comment by nmhung1210** [comment]
> there are still problems with this workflow
> 
> 1. open wallet
> 
> Screen Shot 2019-08-18 at 12 21 50 AM
> 
> 1. call contract in https://member.iotex.io/tools/ERC-20
> 
> Screen Shot 2019-08-18 at 12 23 47 AM
> 
> 1. continue to unlock the wallet. create a new wallet or import one.
> 
> Screen Shot 2019-08-18 at 12 21 34 AM
> 
> the values above are incorrect. they should be like
> 
> Screen Shot 2019-08-18 at 12 21 42 AM

Resolved.

File: config/default.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ module.exports = {
       "https://iotexscan.io/",
       "https://testnet.iotexscan.io/",
       "https://analytics.iotexscan.io/",
-      "ws://localhost:64102/",
+      "wss://localhost:64102/",
```
Comment (puncsky) [inline config/default.js]: wss://local.get-scatter.com:64102

File: src/shared/wallet/get-antenna.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ export function getAntenna(): Antenna {
     injectedWindow.antenna = new Antenna("/iotex-core-proxy", {
       ...(USE_WS_SIGNER
         ? {
-            signer: new WsSignerPlugin("ws://localhost:64102/")
+            signer: new WsSignerPlugin("wss://localhost:64102/")
```
Comment (puncsky) [inline src/shared/wallet/get-antenna.ts]: wss://local.get-scatter.com:64102

**Comment by nmhung1210** [comment]
> after the tx is broadcast, it will not show the broadcast status screen

resolved

**Pull request merged**